components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -5069,6 +5069,40 @@ public void deleteApplication(Application application) throws APIManagementExcep
                 subscriptions.add(rs.getInt("SUBSCRIPTION_ID"));
             }
 
+            prepStmtGetConsumerKey = connection.prepareStatement(getConsumerKeyQuery);
+            prepStmtGetConsumerKey.setInt(1, application.getId());
+            rs = prepStmtGetConsumerKey.executeQuery();
+            List<String> consumerKeys = new ArrayList<>();
+
+            deleteDomainApp = connection.prepareStatement(deleteDomainAppQuery);
+            while (rs.next()) {
+                String consumerKey = rs.getString("CONSUMER_KEY");
```

Comment:
You can use APIConstants.FIELD_CONSUMER_KEY

Reply:
Fixed